src/directives/layertree.js

```diff
@@ -4,15 +4,25 @@ goog.provide('ngeo.layertreeDirective');
 goog.require('ngeo');
 
 
+/**
+ * Unique template path for the layertree;
+ * @type {string|undefined}
+ */
+ngeo.layertreeTemplateUrlValue = undefined;
```

Comment:
I don't understand, why don't you just overright the `ngeoLayertreeTemplateUrl` value ?